unity-environment/Assets/ML-Agents/Scripts/ExternalCommunicator.cs

```diff
@@ -30,9 +30,11 @@ public class ExternalCommunicator : Communicator
     List<float> concatenatedRewards = new List<float>(32);
     List<float> concatenatedMemories = new List<float>(1024);
     List<bool> concatenatedDones = new List<bool>(32);
+    List<bool> concatenatedMaxes = new List<bool>(32);
```

Comment:
This are set as sane defaults. Perhaps we can do a `public const int DEFAULT_NUM_AGENTS = 32` before hand?